provider/storage_benchmark.py

```diff
@@ -270,23 +280,21 @@ def __refresh_session(self, *args, **kwargs):
 
 class IozoneLinuxCfg(object):
     def __init__(self, params, session):
-        version = params.get('iozone_version', 'iozone3_483')
-        self.download_url = ('http://www.iozone.org/src/current/%s.tar' % version)
-        self.download_path = os.path.join('/home', 'iozone.tar')
-        self.iozone_dir = os.path.join('/home/iozone_inst', version)
+        iozone_pkg = params.get("iozone_pkg", 'iozone/iozone3_487.tar.bz2')
```

Comment:
Prefer only  use package name `iozone3_487.tar.bz2`  if define `iozone_pkg`. The folder `iozone` is fixed.

Reply:
Thanks, have updated.